klt-cert-manager/chart/Chart.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v2
+name: keptn-cert-manager
+description: A Helm chart for Keptn Certificate Manager, a subproject of Keptn
+icon: "https://raw.githubusercontent.com/cncf/artwork/master/projects/keptn/icon/color/keptn-icon-color.svg"
+home: https://keptn.sh
+sources:
+  - "https://github.com/keptn/lifecycle-toolkit"
+keywords:
+  - cloud-native
+  - lifecycle
```

Comment:
should we remove all keywords that have to do with lifecycle? This guy actually only does certificates

Reply:
agree